drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -4181,6 +4181,23 @@ RenderingDevice::FramebufferFormatID RenderingDeviceVulkan::framebuffer_get_form
 	return framebuffer->format_id;
 }
 
+bool RenderingDeviceVulkan::framebuffer_is_valid(RID p_framebuffer) const {
+	_THREAD_SAFE_METHOD_
+
+	const Framebuffer *framebuffer = framebuffer_owner.get_or_null(p_framebuffer);
```

Comment:
If this function is the only reason to make the RID owner `mutable`, I'd rather do one of the following:
a) Go filthy locally, with a `const_cast` here.
b) Even better, use `RID_Owner::owns()`, which would require to make it `const`.

Reply:
You are right, I forgot we have owns(), will change it. Thanks!

Reply:
It seems owns was not const, so I had to fix this.